src/main/java/uk/gov/justice/hmpps/prison/service/BookingService.java

```diff
@@ -805,7 +805,7 @@ public void updateLivingUnit(final Long bookingId, final String livingUnitDescri
     }
 
     @Transactional
-    @VerifyBookingAccess
+    @VerifyBookingAccess(overrideRoles = {"SYSTEM_USER", "UNILINK"})
```

Comment:
I'm a bit dubious about whether this is needed, but have left it in anyway.  It is only called from `saveAndReturnCellMoveResult`, which in turn is only called from either `moveToCell` or `moveToCellSwap` that are themselves protected